CONTRIBUTING.md

```diff
@@ -42,12 +47,18 @@ and additionally the following rules:
 * Use `my ($self) = @_;` for parameter parsing in methods when accessing the
   `$self` object. Do not parse any parameter if you do not need any.
 * [DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself)
-* Every pull request is tested by the travis CI by calling the equivalent of
-  `make test` on the tests. It is recommended to call `tools/tidy` or
-  `make tidy` locally to fix the style of your changes before providing a
-  pull request. Call `make test` to conduct all tests.
 
+### Preparing a new Pull Request
+* All code needs to be tidy, for this use `make prepare` the first time you
+  set up your local environment, use `make tidy` or `tools/tidy` locally to
+  ensure your new code adheres to our coding style.
+* Every pull request is tested by the travis CI for different perl versions,
+  if something fails, run `make test` (don't forget to `make prepare` if your setup is new)
+  but the travis results are available too, in case they need to be investigated further
+* Whenever possible, [provide a verification run][1] of a job that runs the code [provided in the pull request][2]
+
+Also see the [DoD/DoR][2] as a helpful (but not mandatory) guideline for new contributions.
+[1]: https://open.qa/docs/#_cloning_existing_jobs_openqa_clone_job
+[2]: https://progress.opensuse.org/projects/openqatests/wiki/Wiki#Definition-of-DONEREADY
+[3]: https://open.qa/docs/#_triggering_tests_based_on_an_any_remote_git_refspec_or_open_github_pull_request
```

Comment:
Good idea to mention this but do you reference it in the text? Couldn't find it

Reply:
Good catch! it's fixed now